### DISCUSSION TITLE
Important Instruction about Phase 3

### DISCUSSION THREAD

**Discussion by zhoujingbo** [PaddlePaddle/PaddleSpatial#266] [BaiduKDDCup2022]
From July 16th 00:01 AM to July 18th 11: 59 PM (AOE time), we will go to Phase 3 of the Baidu KDD Cup. During this phase, the test data will be updated. Each team can submit its prediction model three times per day. During this phase, the organizer team will not answer any questions about submission errors (except for the Pending Timeout error) in the discussion forum. The leaderboard in Phase 3 will determine the candidate awardee teams.

**Comment by luckyyangrun** [comment]
Phase 3 with an open leaderboard has little significance. According to these criteria, the Baidu KDD CUP 2022 is only a two-day, over-fitting competition.

**Reply by makwingchi** [reply]
lol
